README.md

```diff
@@ -198,6 +202,42 @@ IBIS_SCALAR = {
 }
 ```
 
+# Generating Substrait from Adhoc SQL/Ibis
+The CLI tool for generating substrait plans from adhoc SQL queries and Ibis expression
+is located in the `substrait_consumer/tests/adhoc` directory.  The SQL queries should be 
+written using the same TPCH data used in the integration tests.  This tool will generate 
+the substrait plans for each supported producer and run that plan against all supported consumers.
+
+## How to Use
+Copy your SQL query into `substrait_consumer/tests/adhoc/query.sql`.  
+Copy your Ibis expression into `substrait_consumer/tests/adhoc/ibis_expr.py`
+
+*Note: Do not modify line 1 in ibis_expr.py
+
+Run using the following command:
+```commandline
+cd substrait_consumer/tests/adhoc
+pytest test_adhoc_expression.py
+```
+
+Save the substrait plans by specifying the `--saveplan` option.
+```commandline
+pytest --saveplan test_adhoc_expression.py
+```
+Plans will be saved as {producer_name}_substrait.json
+```commandline
+ls *.json
+DuckDBProducer_substrait.json	IbisProducer_substrait.json	IsthmusProducer_substrait.json
+```
+
+If you want to only generate plans from a specific producer or run the plans against a
+specific consumer, you can also use the `--producer` and `--consumer` options.
```

Comment:
Does this also run the plans against the consumers?  The rest of the description only talked about producing plans.

Reply:
Yes it does.  I'll update the description.